contracts/Genesis.sol

```diff
@@ -81,30 +83,44 @@ contract Genesis is ERC721Pausable, VRFConsumerBase, Ownable {
         return baseTokenURI;
     }
 
+    function isRevealed() internal view returns (bool revealed) {
+        return bytes(baseTokenURI).length > 0;
+    } 
+
     /**
      * Setters
      */
-    function setBaseURI(string memory _baseTokenURI) public onlyOwner {
+    function setBaseURI(string memory _baseTokenURI) external onlyOwner {
         baseTokenURI = _baseTokenURI;
     }
-    function setWhiteListMerkleTreeRoot(bytes32 _whiteListMerkleTreeRoot) public onlyOwner {
+    function setWhiteListMerkleTreeRoot(bytes32 _whiteListMerkleTreeRoot) external onlyOwner {
         whiteListMerkleTreeRoot = _whiteListMerkleTreeRoot;
     }
-    function setFreeMintMerkleTreeRoot(bytes32 _freeMintMerkleTreeRoot) public onlyOwner {
+    function setFreeMintMerkleTreeRoot(bytes32 _freeMintMerkleTreeRoot) external onlyOwner {
```

Comment:
good change!

Reply:
check this out https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices

Reply:
yea saw this, good call